MSAL/src/MSALDeviceInformation.m

```diff
@@ -65,6 +78,11 @@ - (NSString *)msalDeviceModeString
     }
 }
 
+- (void) initExtraDeviceInformation:(MSIDDeviceInfo *)deviceInfo
+{
+    [_extraDeviceInformation setValue:deviceInfo.ssoExtensionMode == MSIDSSOExtensionModeFull ? @"Yes" : @"No" forKey:MSAL_DEVICE_INFORMATION_SSO_EXTENSION_FULL_MODE_KEY];
```

Comment:
nit: any reason we're using @"Yes" and @"No" as strings? Can we just pass @NO and @YES? Or you would like to keep it as <NSString, NSString>?

Reply:
Just for readability as we did for the sso extension response. But I am fine if we just use NSNumber. Let me change it.

Reply:
added comment to the function above to emphasize that both key and value should be string :)